From afe34a2ea5bb35985fe5134f5ee8f9a672c19510 Mon Sep 17 00:00:00 2001
From: Naphtali Sprei <nsprei@redhat.com>
Date: Wed, 7 Apr 2010 16:44:16 -0300
Subject: [PATCH 05/21] read-only: No need anymoe for bdrv_set_read_only

RH-Author: Naphtali Sprei <nsprei@redhat.com>
Message-id: <1270658667-14294-6-git-send-email-nsprei@redhat.com>
Patchwork-id: 8440
O-Subject: [RHEL6 kvm PATCH v3 rebase 05/16] read-only: No need anymoe for
	bdrv_set_read_only
Bugzilla: 537164
RH-Acked-by: Daniel P. Berrange <berrange@redhat.com>
RH-Acked-by: Christoph Hellwig <chellwig@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=537164
upstream commit: 37226ad946597126f5bce36f75f818b12fe4b5b8

Signed-off-by: Naphtali Sprei <nsprei@redhat.com>
---
 block.c |    7 -------
 block.h |    1 -
 2 files changed, 0 insertions(+), 8 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 block.c |    7 -------
 block.h |    1 -
 2 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/block.c b/block.c
index 955cd67..2956e4a 100644
--- a/block.c
+++ b/block.c
@@ -1002,13 +1002,6 @@ int bdrv_is_read_only(BlockDriverState *bs)
     return bs->read_only;
 }
 
-int bdrv_set_read_only(BlockDriverState *bs, int read_only)
-{
-    int ret = bs->read_only;
-    bs->read_only = read_only;
-    return ret;
-}
-
 int bdrv_is_sg(BlockDriverState *bs)
 {
     return bs->sg;
diff --git a/block.h b/block.h
index 313b4f3..31e18f5 100644
--- a/block.h
+++ b/block.h
@@ -153,7 +153,6 @@ int bdrv_get_type_hint(BlockDriverState *bs);
 int bdrv_get_translation_hint(BlockDriverState *bs);
 int bdrv_is_removable(BlockDriverState *bs);
 int bdrv_is_read_only(BlockDriverState *bs);
-int bdrv_set_read_only(BlockDriverState *bs, int read_only);
 int bdrv_is_sg(BlockDriverState *bs);
 int bdrv_enable_write_cache(BlockDriverState *bs);
 int bdrv_is_inserted(BlockDriverState *bs);
-- 
1.7.0.3

